src/TabProps.tsx

```diff
@@ -0,0 +1,55 @@
+import { buildQueryExpression, TabOptions, TabProps } from "@coveo/headless";
```

Comment:
I'm not sure if we want/need putting props in a separate file.

I know, it's a matter of preference and/or project conventions. It's not a technical requirement to have props within the component. 🙏

I looked through a couple of components in the UI Kit repository to recall how our fellow devs handle the props. They put them inside components. So should we, I think 🤔

Thoughts? @jpmarceau @eguyon-coveo @srosenberg-coveo @spiracha-coveo

Reply:
I put the props in a different file to improve readability, but if the other teams put them within the component, I think it would be a good idea to do the same